data-es6.js

```diff
@@ -1059,6 +1082,28 @@ exports.tests = [
       },
     },
     {
+      name: 'for-loop iteration scope',
```

Comment:
V8 doesn't fail this, though, as you can see from the `chrome49` entry. :) AFAIK only Firefox & Safari do (on the browser side) and it's already fixed in WebKit Nightly.